po/ko.po

```diff
@@ -6217,17 +6217,17 @@ msgstr "--activation-key 옵션에는 --org를 지정해야 합니다."
 # translation auto-copied from project subscription-manager, version 1.11.X, document keys
 #: src/subscription_manager/migrate/migrate.py:955
 msgid "The --servicelevel and --no-auto options cannot be used together."
-msgstr "--servicelevel 및 --no-auto 옵션은 함께 사용할 수 없습니다."
+msgstr "--servicelevel 및 --no-auto 옵션은 함께 사용 할 수 없습니다."
 
 #: src/subscription_manager/migrate/migrate.py:958
 msgid "The --remove-rhn-packages and --keep options cannot be used together."
-msgstr "--remove-rhn-packages 및 --keep 옵션은 동시에 사용할 수 없습니다."
+msgstr "--remove-rhn-packages 및 --keep 옵션은 동시에 사용 할 수 없습니다."
 
 #: src/subscription_manager/migrate/migrate.py:961
 msgid ""
 "The --remove-rhn-packages option must be used with --registration-state="
 "purge."
-msgstr "--remove-rhn-packages 옵션은 --registration-state=purge로 사용해야 합니다."
+msgstr "--remove-rhn-packages 및 --keep 옵션은 동시에 사용 할 수 없습니다."
```

Comment:
The `--registration-state=` CLI option is not included in translated string. I can see the `--keep` CLI option in translated string.